src/main/resources/sam.conf

```diff
@@ -3,6 +3,9 @@ akka {
     server {
       idle-timeout = 180 s
       request-timeout = 60 s
+      parsing {
+        max-uri-length = 16k
```

Comment:
Added because I was getting `URI length exceeds the configured limit of 2048 characters` in Sam Swagger as my rtitle@quality.firecloud.org. We have made a similar change to Leo and Orch (see https://github.com/broadinstitute/terra-helmfile/pull/5432).